### DISCUSSION TITLE
심사 관련 문의

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#582] [5. Q&A]
안녕하세요 

좀비를 테마로 해서 브롤스타즈와 같은 대전 슈팅 액션 게임을 만들고 있는데요 
![image](https://user-images.githubusercontent.com/50188795/161673737-332fbb45-da0b-47d2-980b-67c6f70c7789.png)

총, 총알과 같은 오브젝트들이 등장 합니다 
총은 다음과 같은 에셋을 사용해 플레이어가 직접 들고 쏘는 모션이 들어갑니다 
![image](https://user-images.githubusercontent.com/50188795/161674084-0f0c4417-a352-4598-b7f4-e1ea72142a22.png)

실제로 총알이 보이지는 않지만 이펙트로 처리 할 예정입니다 
다음과 같이요 
![image](https://user-images.githubusercontent.com/50188795/161673915-eb9ce049-1ec7-4ae5-be9f-50528f21d7fa.png)

좀비는 캐주얼하게 다음과 같은 에셋을 사용해 제작중입니다 
![image](https://user-images.githubusercontent.com/50188795/161673991-00b2ffd9-3274-4a72-b831-0ac07345acd6.png)

위와 같은 게임을 기획하고 만들어나가고 있는데 심사 기준이 궁금 합니다 

어떤 글을 보니깐 총알, 총, Gun, Bullet과 같은게 들어가면 리젝 사유가 되었다는 글을 봐서요 

궁금 합니다

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 리젝 사유는 아니지만 제페토가 전연령이 접근 가능한 앱인만큼
지나치게 폭력적이거나 사회적으로 문제가 될 소재는 피하는 것을 추천 드립니다.
@smilejsu82 님께서 예시로 올려주신 부분은 문제가 되지는 않으나 가급적 총, 무기류의 언급은 지양 부탁 드립니다.
아래 심사 가이드 참고 부탁 드립니다. 

https://studio.zepeto.me/kr/guides/world-guidelines

**Reply by smilejsu82** [reply]
답변 감사합니다 ~ :)

**Reply by Andwhy81kr** [reply]
어엇....저는 제목에 "총알(bullet)"이란 단어가 들어간 이유로 리젝 당한적이 있는데;;;(실제로 제목변경하고 바로 통과됨..)
이건 심사 담당자 마다 기준이 다를까요??
(몇일뒤...Bullet이라는 이름의 월드가 나오는 거보고 좀 황당스럽긴하더군요..)

**Reply by Zepeto-TechEvangelist** [reply]
단어 언급보다도 실제 무기 관련 컨텐츠가 들어갔는지가 핵심입니다.  
공통 가이드라인 1.2 항목 및 커뮤니티 가이드라인 무기 항목 참고 부탁 드립니다.
이후로도 무기에 대한 언급이나 콘텐츠 포함은 가급적 지양 부탁드립니다. 

https://studio.zepeto.me/kr/guides/common-guidelines

https://support.zepeto.me/hc/ko/articles/900005448466-%EC%BB%A4%EB%AE%A4%EB%8B%88%ED%8B%B0-%EA%B0%80%EC%9D%B4%EB%93%9C%EB%9D%BC%EC%9D%B8
